morpheus/modules/databricks_deltalake_writer.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) 2022-2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+import mrc
+import pandas as pd
+from mrc.core import operators as ops
+
+import cudf
+
+from morpheus.utils.module_utils import register_module
+from morpheus.config import Config
+from morpheus.messages import MessageMeta
+from morpheus.utils.module_ids import DATABRICKS_DELTALAKE_WRITER
+from morpheus.utils.module_ids import MORPHEUS_MODULE_NAMESPACE
+
+logger = logging.getLogger(__name__)
+
+IMPORT_EXCEPTION = None
+IMPORT_ERROR_MESSAGE = "DataBricks writer module requires the databricks-connect package to be installed."
+
+try:
+    from databricks.connect import DatabricksSession
+    from pyspark.sql import types as sql_types
+except ImportError as import_exc:
+    IMPORT_EXCEPTION = import_exc
```

Comment:
Is 'IMPORT_EXCEPTION' used anywhere? If we're not handling the failure to import via some kind of work around, then lets log an error here and re-raise the ImportErrror